das-dashboard/backend/shared/internal/web_configuration.py

```
@@ -41,90 +45,163 @@ def load_user_profile(self):
         try:
             with open(PROFILE_PATH, "r") as f:
                 self.user_profile = json.load(f)
-        except:
+        except Exception:
```

Pylint (4.0.6)

[warning] Catching too general exception Exception

(W0718)